db/migrate/20220120124837_create_profiles.rb

```diff
@@ -8,15 +8,15 @@ def change
       t.string :name, null: false
       t.string :patronymic
       t.date :birthday, null: false
-      t.string :email, null: false
+      t.references :user, null: false, index: { unique: true }
```

Comment:
same here. Create new migration. BTW you can find migrations which were RUN for you local DB in "schema_migrations" table.